gpiozero/pins/data.py

```diff
@@ -573,10 +573,12 @@ def _parse_pi_revision(revision):
             '3B': True,
             }.get(model, False)
         csi = {
-            'Zero': 0 if pcb_revision == '0.0' else 1,
+            'Zero': 0 if pcb_revision == '1.0' else 1,
             'CM':   2,
             }.get(model, 1)
-        dsi = csi
+        dsi = {
+            'Zero': csi if pcb_revision == '1.0' else 0,
```

Comment:
Urgh ... not thinking straight - bouncing between two branches now (a cherry picked one and the servo one)! Thanks for the catch :)